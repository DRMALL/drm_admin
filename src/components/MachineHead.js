


import React from 'react'
import { Link } from 'react-router'
import Icon from 'antd/lib/icon'
import Input from 'antd/lib/input'
import searchMachine from '../actions/searchMachine'

const machineHead = () => (
  <div className='machine-head' >
    <div className='machine-head-left' >设备管理</div>
    <Input.Search className='machine-head-input' style={{ width: 300 }}
           placeholder='请输入关键词搜索设备' onChange={ searchMachine } />
    <Link to='/machine/new' className='machine-head-right' >
      <div className='machine-head-right-block' ><Icon type="plus" /></div>
      <div className='machine-head-right-text' >添加新设备</div>
    </Link>
  </div>
  )

export default machineHead

