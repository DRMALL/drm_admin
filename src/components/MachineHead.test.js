

import React from 'react'
import MachineHead from './MachineHead'
import renderer from 'react-test-renderer'

it('MachineHead snapshot test', () => {
  const tree = renderer.create(<MachineHead />)
  expect(tree).toMatchSnapshot()
})
