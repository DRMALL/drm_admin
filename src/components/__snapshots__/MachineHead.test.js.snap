// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MachineHead snapshot test 1`] = `
<div
  className="machine-head"
>
  <div
    className="machine-head-left"
  >
    设备管理
  </div>
  <span
    className="ant-input-affix-wrapper"
    style={
      Object {
        "width": 300,
      }
    }
  >
    <input
      className="ant-input ant-input-search machine-head-input"
      disabled={false}
      onChange={[Function]}
      onKeyDown={[Function]}
      placeholder="请输入关键词搜索设备"
      style={null}
      type="text"
    />
    <span
      className="ant-input-suffix"
    >
      <i
        className="anticon anticon-search ant-input-search-icon"
        onClick={[Function]}
      />
    </span>
  </span>
  <a
    className="machine-head-right"
    onClick={[Function]}
    style={Object {}}
  >
    <div
      className="machine-head-right-block"
    >
      <i
        className="anticon anticon-plus"
      />
    </div>
    <div
      className="machine-head-right-text"
    >
      添加新设备
    </div>
  </a>
</div>
`;
